src/Agent.Service/Windows/AgentService.cs

```diff
@@ -50,6 +50,7 @@ protected override void OnStart(string[] args)
                                 lock (ServiceLock)
                                 {
                                     AgentListener = CreateAgentListener();
+                                    WriteInfo(string.Format("Agent.Listner.exe arguments - {0}", AgentListener.StartInfo.Arguments));
```

Comment:
not sure how useful this trace will be? might just remove it, since the agent.listener will trace it out anyway.